### DISCUSSION TITLE
Make a beginners guide

### DISCUSSION THREAD

**Discussion by mscroggs** [jorgensd/dolfinx_docs#4] [Ideas]
I would be very useful if we had a beginner's guide to using DOLFINx, or maybe just split this pages into introductory and advanced usage (eg mesh creation and io is introductory, custom mesh partitioning is advanced)

**Comment by jorgensd** [comment]
I figured the original tutorial is the beginners guide (most people should get by with importing their mesh with meshio or gmshio).
My original idea was this that this documentation would go into more depth than what any other docs has ever done, covering the nitty gritty details and subtleties that we know and understand (like dofmap block size, index_map_bs).
